contracts/implementation/PoolKeeper.sol

```diff
@@ -162,6 +170,30 @@ contract PoolKeeper is IPoolKeeper, Ownable {
         }
     }
 
+    /**
+     * @notice Called by keepers to perform an update on multiple pools
+     * @param pools A tightly packed bytes array of LeveragedPool addresses to be upkept
+     *     20 bytes       20 bytes       20 bytes     ...
+     * | pool address | pool address | pool address | ... |
```

Comment:
I think the formatting needs improvement here (although I appreciate it's difficult inline within a doc comment).

Maybe added borders on both the top and bottom of the table?

Reply:
It's not really a table, but an array visualisation but yeah can do